huaweicloud/services/lts/resource_hauweicloud_lts_struct_template.go

```diff
@@ -0,0 +1,309 @@
+package lts
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/internal/entity"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/internal/httpclient_go"
+	"io/ioutil"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func ResourceLtsStruct() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceLtsStructTemplateCreate,
+		ReadContext:   resourceLtsStructTemplateRead,
+		DeleteContext: resourceLtsStructTemplateDelete,
+		UpdateContext: resourceLtsStructTemplateUpdate,
+
+		Importer: &schema.ResourceImporter{
+			StateContext: ltsStructResourceImportState,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"content": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"log_group_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"log_stream_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"project_id": {
```

Comment:
the `project_id` is defined as a parameter, but not d.Get("project_id") was found